src/tests/ftest/object/same_key_different_value.py

```diff
@@ -185,8 +185,9 @@ def test_array_to_single_value(self):
                 for j in range(3):
                     if (array_value_data[j][0:length-1] !=
                             read_back_data[j][0:length-1]):
-                        print("Written Data: {}".format(array_value_data[j]))
-                        print("Read Data: {}".format(read_back_data[j]))
+                        self.log.info(
+                            "Written Data: {}".format(array_value_data[j]))
+                        self.log.info("Read Data: {}".format(read_back_data[j]))
```

Comment:
(pylint-logging-format-interpolation) Use lazy % formatting in logging functions